test/utils/HyperdriveTest.sol

```diff
@@ -31,7 +31,7 @@ contract HyperdriveTest is IHyperdriveEvents, BaseTest {
     uint256 internal constant MINIMUM_SHARE_RESERVES = ONE;
     uint256 internal constant MINIMUM_TRANSACTION_AMOUNT = 0.001e18;
     uint256 internal constant CHECKPOINT_DURATION = 1 days;
-    uint256 internal constant POSITION_DURATION = 365 days;
+    uint256 internal constant POSITION_DURATION = 15 days;
```

Comment:
Woah, why are we changing this globally? We just want to change this for the test that you're using, not for all of the tests.

Reply:
Oh sorry meant to undo that, was sanity checking the interest accrued.